library.sh

```diff
@@ -412,26 +412,34 @@ function start_latest_knative_serving() {
 # Install Knative Eventing in the current cluster.
 # Parameters: $1 - Knative Eventing crds manifest.
 #             $2 - Knative Eventing core manifest.
+#             $3 - Knative Eventing in memory channel manifest.
+#             $4 - Knative Eventing mt channel broker manifest.
 function start_knative_eventing() {
   header "Starting Knative Eventing"
   subheader "Installing Knative Eventing"
   echo "Installing Eventing CRDs from $1"
   kubectl apply -f "$1"
   echo "Installing Eventing core components from $2"
   kubectl apply -f "$2"
+  echo "Installing Eventing in memory channel from $3"
+  kubectl apply -f "$3"
+  echo "Installing Eventing mt channel broker from $4"
+  kubectl apply -f "$4"
   wait_until_pods_running knative-eventing || return 1
 }
 
 # Install the stable release Knative/eventing in the current cluster.
 # Parameters: $1 - Knative Eventing version number, e.g. 0.6.0.
 function start_release_knative_eventing() {
   start_knative_eventing "https://storage.googleapis.com/knative-releases/eventing/previous/v$1/eventing-crds.yaml" \
-    "https://storage.googleapis.com/knative-releases/eventing/previous/v$1/eventing-core.yaml"
+    "https://storage.googleapis.com/knative-releases/eventing/previous/v$1/eventing-core.yaml" \
+    "https://storage.googleapis.com/knative-releases/eventing/previous/v$1/in-memory-channel.yaml" \
+    "https://storage.googleapis.com/knative-releases/eventing/previous/v$1/mt-channel-broker.yaml"
```

Comment:
I wonder why there were not there - I guess I lack context here

Reply:
@matzew : I raised this https://github.com/knative/hack/pull/24 yesterday with only -core and -crds and missed the imc and mt-broker, fixing that with this follow up.

Reply:
but since `eventing.yaml` sums that together and is back now (https://github.com/knative/eventing/pull/4546) , I wonder if we should (revert) use that instead of these 4 yamls.